organizations/tests.py

```diff
@@ -91,3 +91,130 @@ def test_updating_organization_recreates_user_instance_cache(self):
         self.assertEqual(
             cache.get(cache_key_name)["organizations"][0]["name"], org_new_name
         )
+
+    def test_settings_support_only_put_method(self):
+        """ /settings/ action only supports PUT method """
```

Comment:
i think we have started the right thing of not unnecessarily writing docstrings for tests when the function name makes it obvious. so, let's remove this?